Makefile

```diff
@@ -500,7 +500,9 @@ engine_unit_test: check_failpoint_ctl
 	$(call run_engine_unit_test,$(ENGINE_PACKAGES))
 
 engine_integration_test: check_third_party_binary_for_engine
```

Comment:
Does it mean we can no longer see the output log in the `action` page directly?
![image](https://user-images.githubusercontent.com/17703670/187581017-59a416b9-da79-4174-8a8e-960a115b5049.png)

Reply:
This pr does not affect tests in Github Action.

Reply:
After moving log to file, do we need to go to the `Artifect` page to get the log, or we can see the log in the first-tier page directly?

Reply:
Since jenkins pipeline would lost some logs, we save output of engine_integration_tests to file.
--- For the user, what happens after this pr?

Reply:
If the log is missing, which means neither `Run all test success` nor `[Makefile:503: engine_integration_test] Error 1` occur in the output, you can find the full output in the Artifect page.

Reply:
you mean `either ... or` ?